### DISCUSSION TITLE
Upgrade Procedure

### DISCUSSION THREAD

**Discussion by matias53** [petrleocompel/hikaxpro_hacs#142] [Q&A] [answered]
Hi, I was wondering what is the procedure to upgrade the version of hikaxpro_hacs? Is it anywhere in the documentation?

**Comment by petrleocompel** [comment]
What upgrade procedure you mean? 
If you have it installed via HACS just update via HACS.

**Comment by petrleocompel** [comment, marked as answer]
https://hacs.xyz/docs/use/update/#to-update-a-downloaded-repository
